app/components/tracks/TracksController.js

```diff
@@ -31,6 +33,57 @@ tracksModule.controller('TracksController',
                 }, function (error) {
                     console.error(error);
                 });
+
+            ApiJsonFactory.getJson('sessions')
+                .then(function (response) {
+                    tc.Sessions = response.data.sessions;
+                    $sessionStorage.sessions = tc.Sessions;
+                }, function (error) {
+                    console.error(error);
```

Comment:
I suggest reading http://stackoverflow.com/questions/14086675/which-browsers-support-console-log . Your code will not work on IE9. I do not believe breaking the website for millions of potential users is worth it, when a single if statement can fix the problem. Let's support as many browsers as we reasonably can. As a data point, I am using IE9 right now (the horror!), because I am in the hospital and cannot use my own computer. Github made sure their site still works on IE9 and so should we.